stake-pool/cli/src/main.rs

```diff
@@ -852,96 +856,25 @@ fn command_deposit_sol(
 
 fn command_list(config: &Config, stake_pool_address: &Pubkey) -> CommandResult {
     let stake_pool = get_stake_pool(&config.rpc_client, stake_pool_address)?;
+    let reserve_stake_account_address = stake_pool.reserve_stake.clone().to_string();
+    let total_lamports = stake_pool.total_lamports.clone();
+    let last_update_epoch = stake_pool.last_update_epoch.clone();
     let validator_list = get_validator_list(&config.rpc_client, &stake_pool.validator_list)?;
+    let max_number_of_validators = validator_list.header.max_validators.clone();
+    let current_number_of_validators = validator_list.validators.clone().len();
```

Comment:
Are all these clones necessary? I think most or all of these types impl Copy.

Reply:
You right it's not. removed